lib/lib.sh

```diff
@@ -96,8 +96,6 @@ get_meta() {
             return 0
         fi
         attempts+=1
-        backoff=$((attempts*ms_per_backoff))
-        sleep $((backoff/1000)).$((backoff%1000))
```

Comment:
This has the same latent bug as line 100, the fractional part will only format correctly if the tenths digit is non-zero.